hyperapp.d.ts

```diff
@@ -94,12 +116,15 @@ export interface View<State, Actions> {
  * @returns The actions wired to the application.
  * @memberOf [App]
  */
-export function app<State, Actions>(
+export function app<
+  State extends StateType,
+  AppActions extends ActionsType<State, AppActions>
```

Comment:
@Alber70g Why `AppActions`?

Reply:
Because otherwise it's ambiguous and can Typescript not detect which one to use. This is the generic where 'Actions' is also a type itself